Bio/SeqFeature.py

```diff
@@ -1641,7 +1641,7 @@ def _shift(self, offset):
     def _flip(self, length):
         """Return a copy of the locations after the parent is reversed (PRIVATE).
 
-        Note that the order of the parts is NOT reversed too. Consider a CDS
+        Note that the order of the parts is NOT reversed if strand!= None. Consider a CDS
```

Comment:
How about "Note that the order of the parts is NOT reversed if any of the parts are stranded", and reword the line wrapping?